.github/workflows/release-image.yaml

```diff
@@ -0,0 +1,38 @@
+name: Generate Release Image
+
+on: [pull_request]
+
+jobs:
+  release-image:
+    # Only generate the release image from the PR contents if the PR was in the
+    # monorepo and opened by lit-robot.
+    if: github.repository == 'lit/lit' && github.actor == 'lit-robot'
+
+    name: Generate Release Image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: 'npm'
+          cache-dependency-path: package-lock.json
+
+      - uses: google/wireit@setup-github-actions-caching/v1
+
+      - name: NPM install
+        run: npm ci
+
+      - name: Build release-image script
+        working-directory: packages/internal-scripts
+        run: npm run build
+
+      - name: Create release image
+        run: node_modules/.bin/release-image -m <(node scripts/fetch-release-changelog.js <<< ${{ github.event.pull_request.number }})
```

Comment:
Just curious.. does this not work without `<<<` and reading from `process.argv[2]` within the script?

Reply:
Done! Great suggestion!